index.html

```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
         asymmetric encryption algorithms.
     </p>
 
+  <section id="conformance">
+    <section class="normative">
+        <h2 id="conformance-classes">Conformance Classes</h2>
```

Comment:
It seems odd that `<section id="introduction">` is not set as `<section id="introduction" class="normative">... and then the `class="normative"` removed from the sub-sections it contains.

```suggestion
  <section id="conformance" class="normative">
    <section class="normative">
        <h2 id="conformance-classes">Conformance Classes</h2>
```